openy.install

```diff
@@ -904,18 +914,26 @@ function openy_update_8059() {
   \Drupal::service('module_installer')->uninstall(['location_finder']);
 }
 
-/**
- * Enable Open Y Entity Reference Tweaks.
- */
-function openy_update_8060() {
-  \Drupal::service('module_installer')->install(['openy_er']);
-}
-
 /**
  * Enable Open Y Entity Reference Tweaks.
  */
 function openy_update_8062() {
   \Drupal::service('module_installer')->install(['openy_er']);
+
+  if (\Drupal::service('module_handler')->moduleExists('openy_migrate')) {
+    $modules = [
+      'openy_prgf_event_latest',
+      'openy_prgf_event_listing',
+      'openy_node_event',
+      'openy_demo_nevent',
+    ];
+
+    // Enable modules.
+    \Drupal::service('module_installer')->install($modules, TRUE);
```

Comment:
installation/uninstallation should be in dedicated update_N stability/reliability/memory wise

Reply:
this is the same set of modules as in the openy_update_8053, we just moved it down.